emcc.py

```diff
@@ -3156,8 +3156,6 @@ def process_libraries(libs, lib_dirs, settings_changes, input_files):
     if not found:
       libraries += shared.Building.path_to_system_js_libraries(lib)
 
-  # Certain linker flags imply some link libraries to be pulled in by default.
-  libraries += shared.Building.path_to_system_js_libraries_for_settings(settings_changes)
```

Comment:
I think you can also remove the settings_changes arg from this function too.

Reply:
Thanks, done.

Reply:
Hmm..  you can also just remove Found completely and return on line 3150?

Reply:
Hmm, yeah, but then we'd need to duplicate the `'SYSTEM_JS_LIBRARIES="' + ','.join(libraries) + '"'` line. Not sure which I prefer.

Reply:
ok lgtm